.ado/apple-integration.yml

```diff
@@ -4,15 +4,17 @@ variables:
 trigger:
   branches:
     include:
-      - main
+    # Disable Integration on main, as we are not currently stable
+    # - main
```

Comment:
If we do disable a test, we should have a bug tracking adding it back so it's not lost in the ether